test/taskincludes_2_7_style.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: webservers
+  vars:
+    varset: varset
+  tasks:
+    - include_tasks:
+        file: "{{ varset }}.yml"
+    - include_tasks:
+        file: "directory with spaces/main.yml"
```

Comment:
You only test the code branch that always has `file`. Please also cover the case w/o it.